packages/suite-desktop/src-electron/modules/theme.ts

```diff
@@ -0,0 +1,37 @@
+import { nativeTheme, ipcMain } from 'electron';
+
+import { SuiteThemeVariant } from '@suite-types';
+import { getThemeSettings, setThemeSettings } from '@desktop-electron/libs/store';
+
+const init = () => {
+    const { logger } = global;
+
+    const theme = getThemeSettings();
+    if (theme !== 'system') {
+        logger.info('theme', `Setting title bar theme to ${theme}.`);
+        nativeTheme.themeSource = theme;
+    }
+
+    ipcMain.handle('theme/change', (_, theme: SuiteThemeVariant) => {
+        if (theme === 'light') {
+            logger.info('theme', `Manually setting title bar to light theme.`);
+
+            nativeTheme.themeSource = 'light';
+            setThemeSettings('light');
+        } else {
+            logger.info('theme', `Manually setting title bar to dark theme.`);
+
+            nativeTheme.themeSource = 'dark';
+            setThemeSettings('dark');
```

Comment:
Is it necessary to repeat identical code block for `light`/`dark` mode (and also for `system` mode below)?

Reply:
c0593d9e817a88b8524e9bb6c1f05ac41fa3aecd